doc/orchestrator.rst

```diff
@@ -1,208 +1,685 @@
+  Note that the C/C++/Fortran SmartRedis ``Clients`` are capable of reading cluster configurations
+  from the SmartSim ``Model`` environment and the `cluster` constructor argument does not need to be specified
+  in those ``Client`` languages.
 
-|colo-orc|
+.. note::
+    Since there is only one ``Orchestrator`` launched in the experiment
+    (the standalone ``Orchestrator``), specifying an ``Orchestrator`` `db_identifier`
+    is **not** required when initializing the SmartRedis ``Client``.
+    SmartRedis will handle the connection configuration.
 
-This deployment is designed for highly performant online inference scenarios where
-a distributed process (likely MPI processes) are performing inference with
-data local to each process.
+.. note::
+   To create a SmartRedis ``Client`` connection to the standalone ``Orchestrator``, the ``Orchestrator`` must be launched
+   from within the driver script. You must execute the Python driver script, otherwise, there will
+   be no ``Orchestrator`` to connect the ``Client`` to.
+
+Data Retrieval
+--------------
+To confirm a successful connection to the ``Orchestrator``, we retrieve the tensor set from the ``Experiment`` driver script.
+Use the ``Client.get_tensor()`` method to retrieve the tensor by specifying the name `tensor_1` we
+used in the driver script as input:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_app.py
+    :language: python
+    :linenos:
+    :lines: 7-10
+
+After the ``Model`` is launched by the driver script, the following output will appear in
+`getting-started/model/model.out`::
+
+  Default@17-11-48:The multi-sharded db tensor is: [1 2 3 4]
+
+Data Storage
+------------
+Next, create a NumPy tensor to send to the standalone ``Orchestrator`` using
+``Client.put_tensor(name, data)``:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_app.py
+    :language: python
+    :linenos:
+    :lines: 12-15
+
+We will retrieve `"tensor_2"` in the Python driver script.
+
+.. _standalone_orch_driver_script:
+Experiment Driver Script
+========================
+To run the previous application script, we define a ``Model`` and ``Orchestrator`` within a
+``Experiment`` driver script. Configuring and launching workflow entities (``Model`` and ``Orchestrator``) requires the utilization of
+``Experiment`` class methods. The ``Experiment`` object is intended to be instantiated
+once and utilized throughout the workflow runtime.
 
-This method is deemed ``locality based inference`` since data is local to each
-process and the ``Orchestrator`` is deployed locally on each compute host where
-the distributed application is running.
+In this example, we instantiate an ``Experiment`` object with the name `getting-started`
+and the `launcher` set to `auto`. When using `launcher=auto`, SmartSim attempts to find a launcher on the machine.
+In this case, since we are running the example on a Slurm-based machine,
+SmartSim will automatically set the launcher to `slurm`.
```

Comment:
Replace "In this case, since we are running" to "For example, if this script was run on a Slurm-based"